next.config.mjs

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  experimental: {
+    serverComponentsExternalPackages: ["@node-rs/argon2"],
```

Comment:
As far as I know, this is included in nextjs automatically now: https://github.com/vercel/next.js/issues/63850

Also, be aware of this: https://github.com/pilcrowOnPaper/oslo/pull/69

oslo is removing the password module, which is the reason for this dependency.

Reply:
Okay, saw the issue on vercel and removed the config: https://github.com/complexdatacollective/Studio/pull/13/commits/70c82d366a7eee763321a4fdd145dbb092dac0dd

Reply:
Added back the config because we don't have the latest changes from Next yet